extensions/guacamole-auth-nextcloud/src/main/resources/guac-manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+
+    "guacamoleVersion" : "1.5.4",
```

Comment:
This probably needs to be bumped to `1.5.5`, since that's the current version.

Reply:
I have increased the version to 1.5.5